src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs

```diff
@@ -3426,7 +3426,7 @@ void MethodHasNoAttribute()
             await VerifyAnalyzerAsyncCs(source, s_msBuildPlatforms);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn-analyzers/issues/4920")]
+        [Fact]
         [WorkItem(4920, "https://github.com/dotnet/roslyn-analyzers/issues/4920")]
         public async Task TestTimelyTermination()
```

Comment:
Can you add a test for https://github.com/dotnet/roslyn/issues/51652 as well to make sure this PR fixes both?

Reply:
Doesn't seem to fix that second issue, its still hanging. Will continue to investigate

Reply:
@mavasani Should it be re-opened and moved in this repo for investigation?

Reply:
Test added and now passes. I think it is fine to keep the issue closed.